### PR TITLE
feat : typography 

### PR DESCRIPTION
**Pull request by pankaj-prog**
removed redundant code and added typography
Screenshot 2022-02-12 001532

Screenshot 2022-02-12 001730

Screenshot 2022-02-12 001541
.

### CODE REVIEW

File: src/components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        <section class="component-example-wrapper">
+          <div class="grid-2c grid-wrapper">
+            <div class="grid-item-one">grid-item-one</div>
+            <div class="grid-item-two">grid-item-two</div>
+          </div>
+        </section>
+        <section class="component-code-wrapper">
+          <script type="text/plain" class="line-numbers language-markup">
+            <div class="grid-2c">
+              <div class="..."> ... </div>
```
Comment (sanjitsarkar) [inline src/components/grid/grid.html:105]: I think ... for class name doesn't seem meaningful

**Review by sanjitsarkar — APPROVED** [review]

File: src/components/grid/grid.html (hunk 1 above)
Comment (pankaj-prog) [reply src/components/grid/grid.html:105]: okay i will change it.

File: src/components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.grid-two-columns {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid-three-columns {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (gauravsinhaweb) [inline src/components/grid/grid.css]: use repeat here.

File: src/components/grid/grid.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          </li>
+        </ul>
+      </nav>
+      <main class="site-content">
+        <h1 class="component-name">Grid</h1>
+        <p class="component-description">
+          Grids provide a common graphic language that makes it easier for both
+          designers and developers to work on a website. It helps users to
+          better understand the information as it's shown in a more structured
+          and logical way
```
Comment (gauravsinhaweb) [inline src/components/grid/grid.html]: way,

File: src/components/typography/typography.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
+.h1,
+.h2,
+.h3,
+.h4,
+.h5 {
```
Comment (gauravsinhaweb) [inline src/components/typography/typography.css:5]: add elements also i.e. h1,h2 h3...

File: style.css [hunk 5]
```diff
@@ -180,19 +183,46 @@ body {
...
-  width: fit-content;
-  background-color: var(--BLACK);
-  color: var(--OFFWHITE);
-  padding: 0 0.5em;
+/* Grids demo  */
+.grid-wrapper {
+  width: 100%;
+  max-height: 10rem;
+  background-color: var(--GRAY);
+  grid-gap: 2rem;
```
Comment (gauravsinhaweb) [inline style.css]: use gap instead of grid-gap. It is the older version of gap.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
The UI is clean and elegant. Navbar scroll is so cool. 

Good to merge after changes

File: style.css (hunk 5 above)
Comment (pankaj-prog) [reply style.css]: Sure.

File: src/components/grid/grid.css (hunk 2 above)
Comment (pankaj-prog) [reply src/components/grid/grid.css]: okay

**Pull request merged**